src/shared/components/selected-species/SpeciesLozenge.tsx

```diff
@@ -35,17 +34,18 @@ export type Props = DetailedHTMLProps<
   HTMLButtonElement
 > & {
   species: CommittedItem;
+  view: string;
```

Comment:
The type of this property should be restricted to only the allowed strings (through the union type I suggested for `SpeciesLozengeDisplaySelector`.